### ISSUE TITLE
Hot-reloading not working if assets fingerprinting enabled

### ISSUE THREAD

**Issue by lifeart** [lifeart/ember-ast-hot-load#62]
reproduction:

```js
  let app = new EmberAddon(defaults, {
  // Add options here
    'ember-ast-hot-load': {
      helpers: ["my-zoo"],
      enabled: true
	},
	fingerprint: {
		enabled: false,
		generateAssetMap: false,
		fingerprintAssetMap: false
	}  
  });
```

**Comment by lifeart** [comment]
fix: https://github.com/lifeart/ember-ast-hot-load/pull/61

**Comment by lifeart** [comment]
fixed in `v0.0.79`
